journal/modernized/13-chapter-11.adoc

```diff
@@ -244,8 +254,8 @@ and bringing forth the birth of the morning of his sweet day, through this and +
 nations, to the magnifying, renowning and exalting his blessed name forever,
 is the breathing and travail of your tribulated brother
 
+[.postscript]
```

Comment:
this postscript looks funny but I wasn't sure what to do with it.

Reply:
Yeah... when Jared created the postscript tag I think he was thinking we would use it on the postscript TEXT itself, after a signature, but you're right, it looks funny putting it on the word "Postscript". Tell you what... lets mark the word Postscript with an [.offset] tag instead. That will give it a space above and below the word.

Reply:
👍 Do you want it to look any different way so it stands out more?